src/main/objects/HarmonicFunction.js

```diff
@@ -86,6 +88,10 @@ function HarmonicFunction2(params){
         return res;
     };
 
+    this.countAllComponents = function () {
```

Comment:
Wydaje mi się że w allComponents powinien się wliczać delay... tylko pytanie jak? Na razie w żaden sposób nie oznaczamy czy jest to pierwsza funkcja z dealay'u czy druga, więc nie da się tego określić. Chyba że intencja wprowadzenia tej funckji jest inna niż myśle

Reply:
Chciałem użyć tej funkcji gdzieś, koniec końców nie użyłem i się zacommitowała. Usunąłem.